packages/bootleg-app/README.old.md

```diff
@@ -0,0 +1,26 @@
+<h1 align="center">
```

Comment:
Why keep this one? or the other?

Reply:
`create-react` clobbered this file and named it `.old` I left because I have not gotten around to cleaning it up.  I will.